### ISSUE TITLE
CommandBar: [HighContrast] menu items shifts towards right on hover

### ISSUE THREAD

**Issue by nipunja** [OfficeDev/office-ui-fabric-react#791]
### Bug Report

- __Package version(s)__: latest
- __Browser and OS versions__: Tested on IE11 & might be applicable to other browsers as well

#### Describe the issue:

[HighContrast]Right Shift in menu items of commandBar on hover

#### Actual behavior:

[HighContrast]Right Shift in menu items of commandBar on hover
![image](https://cloud.githubusercontent.com/assets/11870255/21847867/3522e57e-d824-11e6-8efd-fbd5c42dc911.png)

See the highlighted section. Extra border on hover causes a shift of command bar menu items to the right side.

#### Expected behavior:

Shift should not happen.

### If applicable, please provide a codepen repro:
https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/commandbar ( See this in high contrast settings with black background)

**Comment by aleksandrjPersonal** [comment]
Merged
